scripts/microrepl.py

```diff
@@ -165,10 +165,12 @@ def main():
     port = sys.argv[1]
     print('Device path', port)
 
-    term = connect_miniterm(port)
+    
 
     if os.name == 'nt':
-        term.console = Windows10Console()
+        miniterm.console = Windows10Console()
```

Comment:
@jdjjm I don't get how `miniterm.console` is used after that. It seems to me like this statement has no effect on what happens next.

No code in `serial/tools/miniterm.py` actually uses the global (file-level) variable `console`. Instead, instances of the class `Miniterm` use their instance-level attribute `console` via `self.console`.